main.go

```diff
@@ -99,6 +99,11 @@ func wrapper(fn func(w http.ResponseWriter, r *http.Request)) http.HandlerFunc {
 	}
 }
 
+func HealthCheckHandler(w http.ResponseWriter, r *http.Request) {
+	setStrHeader(w, "Content-Type", "text/plain")
+	io.WriteString(w, "WORKING\n")
```

Comment:
I think "ok" would be better for the health check response body.

Reply:
Good ideia, let me change this.